src/qrcode/utils.ts

```diff
@@ -126,9 +127,10 @@ export function* handleSecureSend(
 }
 
 function* extractQRAddressData(qrCode: QrCode) {
-  // Regex matches any 40 hexadecimal characters prefixed with "0x" (case insensitive)
-  if (/^0x[a-f0-9]{40}$/gi.test(qrCode.data)) {
-    qrCode.data = `celo://wallet/pay?address=${qrCode.data}`
+  // strip network prefix if present
+  const qrAddress = qrCode.data.replace(/^.*:/, '')
+  if (isAddress(qrAddress, { strict: false })) {
```

Comment:
...so why wouldn't we want `strict: true`?

Reply:
`Strict: true` would mean that only checksum'd addresses will be valid. I didn't think it wise to add this additional restriction when we don't have any control over the QR codes that are being scanned.

Reply:
I don't think valora address qr codes are checksummed either 😅